README.md

```diff
@@ -10,9 +10,14 @@ Have Docker installed.
 
 ## Clone the repository to your development environment
 
-Follow [these instructions](https://help.github.com/articles/fork-a-repo/)
+Follow [these instructions](https://help.github.com/articles/cloning-a-repository/)
 to get a local version of this repository.
 
+##Create a new branch of the repository
+
+Follow [these instructions](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches)
```

Comment:
Can you clarify that, "when working on a particular change, it is best to work in a local branch"

and then we can add a step, "when you believe your changes are ready, add them to the git workspace with `git add -A`, commit them with `git commit -m "description of change here"`, and then push them using `git push origin head`

Finally, navigate to github.com/heartpoints/heartpoints.org, and look for the prompt to create a new pull request from your newly pushed branch. Here, you can review your changes and if things look good, add any potential code reviewers in the upper right, add an optional explanation in the text box, and click "Create pull request"

You will be taken to the newly created pull request page, where you can see your pull request is automatically tested by the development pipeline. Look for the "Status" or "Details" links on the "Conversation" tab of the pull request to understand what automatic testing jobs are running, and whether they passed or failed (you will see a yellow circle for running, green check for passing, red x for failing)

Once you have passing jobs with green checks and at least one review signoff from a colleague, then, assuming you have the latest changes incorporated into your branch, you will be ready to merge! Click "Squash and Merge" to squash your commits into a single commit for the pull request as a whole, and merge that into master.

At that point, the pipeline will automatically build, test, deploy, and production verify your change. You can validate your change has made it to production by using the chrome developer tools to inspect the http response headers, looking for the `commitSha` header, which should match the sha of your merged PR commit in master.